frontend/src/utils/constants.ts

```diff
@@ -45,6 +45,9 @@ export const getCriteriaName = (t: TFunction, criteria: string) => {
 
 export const YOUTUBE_POLL_NAME = 'videos';
 
+const UID_DELIMITER = '%3A';
+export const UID_YT_NAMESPACE = 'yt' + UID_DELIMITER;
```

Comment:
I would recommend to define the delimiter as `:`, and to URL-encode it strictly when necessary (i.e in URL paths, as the OpenAPI client seems to handle `:` as a special case, although it is a valid character). 
For example it would be incorrect to use the encoded version in a request body.